src/python/pants/backend/shell/lint/shellcheck/rules_integration_test.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2021 Pants project contributors.
```

Comment:
Tests are all new, but mostly copy pasta from the recently rewritten Bandit tests.